WordPress/Sniffs/Security/EscapeOutputSniff.php

```diff
@@ -446,12 +446,13 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 	 *
 	 * @since 3.0.0 Split off from the process_token() method.
 	 *
-	 * @param int $start The position to start checking from.
-	 * @param int $end   The position to stop the check at.
+	 * @param int    $start The position to start checking from.
+	 * @param int    $end   The position to stop the check at.
+	 * @param string $code  Code to use for the PHPCS error.
 	 *
 	 * @return int Integer stack pointer to skip forward.
 	 */
-	protected function check_code_is_escaped( $start, $end ) {
+	protected function check_code_is_escaped( $start, $end, $code ) {
```

Comment:
General question: why not set `'OutputNotEscaped'` as the default value for the new `$code` parameter and only pass it when it is going to be different ?

Reply:
I started with that, but once I found there were some recursive calls that needed handling it seemed safer to be explicit on all the calls.

Reply:
Hmm... I'd prefer to use a default value. If that really wouldn't work, I'd like to suggest adding class constants for the error codes and referring to those constants in the function calls to prevent having the hard-coded codes all over the place with the risk of a typo creeping in.

Reply:
Done, not worth arguing over IMO.